utils/utils.jsx

```diff
@@ -1187,17 +1187,6 @@ export function fileSizeToString(bytes) {
     return bytes + 'B';
 }
 
-// Gets the websocket port to use. Configurable on the server.
```

Comment:
This function wasn't used anywhere.

Reply:
I feel like we probably have a lot of those in this file